src/app/help/index.js

```diff
@@ -0,0 +1,49 @@
+import { actionText, dim } from 'src/utils/log-style'
+import { eslintDoc } from 'src/app/eslint'
+import { stylelintDoc } from 'src/app/stylelint'
+
+function header() {
+  console.log(`\n           .${actionText('\tCode-config CLI')}`)
+  console.log('          / V\\')
+  console.log('        / `  /')
+  console.log('       <<   |\tCommand-line tools to setup and manage')
+  console.log('       /    |\tproject configurations.')
+  console.log('     /      |')
+  console.log('   /        |\tGo Bravely')
+  console.log(' /    \\  \\ /')
+  console.log('(      ) | |')
+  console.log('|   _/_  | |')
+  console.log('_\\______)\\__)\n')
```

Comment:
![Screenshot 2023-03-21 at 15 37 09](https://user-images.githubusercontent.com/3470817/226640328-5908dabe-5756-4e11-babe-10a6183b157b.png)

I think it will depend mostly on how people have their font configuration, hopefully monospace type :)